.github/workflows/build.yml

```diff
@@ -0,0 +1,11 @@
+name: Docker Build
+
+on: push
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Docker image
+      run: docker run --name mysql-db -d ${{ env.IMAGE_REPOSITORY }}:latest -e MYSQL_ROOT_PASSWORD=a029d0df84eb5549c641e04a9ef389e5
```

Comment:
🛑 **gitleaks** has detected a secret with rule-id `` in commit 6ccda22c9a8773a159761dc68f00fdf2e3184f22
If this secret is a true positive, please rotate the secret ASAP and rebase the commit containing the secret.